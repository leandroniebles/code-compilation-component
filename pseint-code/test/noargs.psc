Proceso Prueba
  EsperarTecla
  EsperarTecla;
  EsperarTecla ;
  BorrarPantalla;
  Borrar Pantalla;
  BorrarPantalla ;
  Borrar Pantalla ;
  BorrarPantalla 
  Borrar Pantalla 
  Limpiar Pantalla ;
  LimpiarPantalla ;
  Limpiar Pantalla 
  LimpiarPantalla 
  Limpiar Pantalla;
  LimpiarPantalla;
FinProceso