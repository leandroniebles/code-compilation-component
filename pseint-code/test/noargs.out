***EsperarTecla***
***EsperarTecla***
***EsperarTecla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
***LimpiarPantalla***
